packages/page-contracts/src/Codes/Upload.tsx

```diff
@@ -83,6 +84,18 @@ function Upload ({ onClose }: Props): React.ReactElement {
     abiName && setName(abiName);
   }, [abiName, setName]);
 
+  useEffect((): void => {
+    accountId && hasStorageDeposit &&
+      api.derive.balances
+        .account(accountId)
+        .then(({ freeBalance }) => {
+          // instantiation fails when using the entire free balance
+          const limit = freeBalance.sub(freeBalance.div(new BN(10)));
```

Comment:
You need to substract `value` and the existential deposit here. So that the sending account stays above the existential deposit even after sending the `value` and using all of the storage deposit.